public/css/style.css

```
@@ -52,9 +52,29 @@ main {
     color: var(--text-primary-color);
     margin-bottom: 20px;
 } */
+#race-name {
+}
```

Biome (1.9.4)

[error] An empty block isn't allowed.

Consider removing the empty block or adding styles inside it.

(lint/suspicious/noEmptyBlock)